File: 1072.py
n = int(input())
dentro = fora = 0
for i in range(0, n):
    l = int(input())
    if l >= 10 and l <= 20:
        dentro += 1
    else:
        fora += 1
print(f'{dentro} in')
print(f'{fora} out')

File: 1072.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int dentro = 0, fora = 0;
        for(int i = 0; i < n; i++) {
                int l = input.nextInt();
                if(l >= 10 && l <= 20) {
                        dentro += 1;
                }
                else {
                        fora += 1;
                }
        }
        System.out.println(dentro+" in");
        System.out.println(fora+" out");
}
}
